conda-recipe/meta.yaml

```diff
@@ -17,12 +17,14 @@ requirements:
     - numpy
     - six
     - matplotlib
+    - enaml
```

Comment:
build reqs - Packages required to build the package
run reqs - Packages required to run the packages, (Dependencies)
I think you bubblegum need enaml to build and run?